reactMovieApp/src/api/FetchMovies.js

```diff
@@ -1,26 +1,25 @@
+import { getData } from "./getData";
 import { CONFIG } from "../config";
-import React, { useEffect, useState } from "react";
 import MovieCard from "../components/MovieCard";
+import  React, { useEffect, useState } from "react";
 
 const API_URL = CONFIG.BASE_URL + CONFIG.POPULAR_MOVIE_URL + CONFIG.API_KEY;
```

Comment:
is it fully correct to use react methods in API client file? 
maybe it's better to left here only fetching, like we did it in vanilla app?

Reply:
please move FetchMovies inside feature branch